src/Model.php

```diff
@@ -886,6 +886,8 @@ public function tryLoadBy($field, $value)
      *
      * @return $this
      */
+    public $_dirty_after_reload = [];
```

Comment:
i thought since it's $_xx and we don't want anyone to use it but can't make it "private" due to our other class referring to it and also because our style checkers didn't complain, then we can leave here.